addons/binding/org.openhab.binding.miio/database/yeelink.light.ceiling1.json

```diff
@@ -0,0 +1,79 @@
+{
+  "deviceMapping": {
+    "id": [
+      "yeelink.light.ceiling1"
+    ],
+    "channels": [
+      {
+        "property": "power",
+        "friendlyName": "Power",
+        "channel": "power",
+        "type": "Switch",
+        "refresh": true,
+        "ChannelGroup": "",
+        "actions": [
+          {
+            "command": "set_mode",
```

Comment:
Hi

this should be probably "set_power" as well, because my ceiling lamp is complaining, that the method does not exist:
>  Command added to Queue {"id":XXX61,"method":"set_mode","params":["on"]} -> 192.168.189.31 (
Device: XXXXXXA2 token: XXX Queue: 1)
Received response for XXXXXXA2 type: SET_MODE_BASIC, result: null, fullresponse: {"error":{
"code":-32601,"message":"Method not found."},"id":XXX61}

Reply:
Totally right. Thx